tutorials/inverse/35_dipole_orientations.py

```diff
@@ -12,6 +12,23 @@
 tutorial, we will look at the various options available to restrict the
 orientation of the dipoles and the impact on the resulting source estimate.
 
+.. warning::
+   A common "gotcha!" is that by default, dipole orientation information is discarded
+   in the source estimate. Only the magnitude of the activity is retained. This means
```

Comment:
I think this is true when using `loose != 0`. For fixed ori I think it stays signed even with `pick_ori=None` (the default). But given that `loose="auto"` defaults to `0.2` even for surfaces I think it's okay to delve into these details elsewhere. Thanks @wmvanvliet !

Reply:
You are correct :)